packages/core/src/types/IVolumeInput.ts

```diff
@@ -31,6 +31,8 @@ interface IVolumeInput {
   blendMode?: BlendModes;
   /** Slab thickness of the volume - by default it is 0.05*/
   slabThickness?: number;
+  /** other metadata that is needed for the volume */
```

Comment:
Please change the IVolumeInput to a type - that way you don't need to declare the key function, you can simply create an extended type based on it, and use that.